### DISCUSSION TITLE
store lookup values as tags [Apr 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1838] [Discord Threads]
> 🙋‍♂️ **vishal.gandhi**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### could u mind elaborate a bit?  
#  
> 👨‍🚀 **wingkwong**  
#### for lookup sample, please check out https://docs.nocodb.com/setup-and-usages/lookup  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### i would like to store multiple tags in a row  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### e.g. expense table    
#### Category - Restaurant   
#### Tags - out ; april  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### if i want to query by any of the tag ; i should be able to  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### and these tags could be stored in a master table  
#  
> 👨‍🚀 **wingkwong**  
#### lookup on hasManyCells will be available in the next release  
#  
> 👨‍🚀 **wingkwong**  
#### if i understand correct,    
####    
#### expense table -> lookup table -> tag table   
####    
#### your structure s like this right?  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### lets consider this example    
#### Employee ID | Employee Name | Project (tags)   
#### 8272 | Xyz | 1 2, 3    
#### 92828| abc | 4,5   
####    
#### Project Lookup    
#### Project ID    
#### 1 A    
#### 2 B   
#### 3 C   
#### 4 D   
#### 5 E  
#  
> 👨‍🚀 **wingkwong**  
#### "I would like to store lookup value from other table and fields as tags "   
####    
#### then your lookup column would be either multiSelectCell or hasManyCell  
#  
> 👨‍🚀 **wingkwong**  
#### if it is the first case, the current version supports it  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### any documentation i can refer to  
#  
> 👨‍🚀 **wingkwong**  
#### the one i sent u  
#  
> 👨‍🚀 **wingkwong**  
#### ^  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/964018137731309598)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
